cpp/README.md

```diff
@@ -0,0 +1,47 @@
+# Alluxio C++ API
```

Comment:
Should we add a section describing that the C++ client is implemented through JNI to native Java client? And also describe the estimation for performance impact comparing to using the native Java client directly?